protocol/message_processor_test.go

```diff
@@ -33,22 +34,18 @@ type MessageProcessorSuite struct {
 
 	processor   *messageProcessor
 	tmpDir      string
-	testMessage v1protocol.Message
+	testMessage Message
 	logger      *zap.Logger
 }
 
 func (s *MessageProcessorSuite) SetupTest() {
-	s.testMessage = v1protocol.Message{
-		Text:      "abc123",
-		ContentT:  "text/plain",
-		MessageT:  "public-group-user-message",
-		Clock:     154593077368201,
-		Timestamp: 1545930773682,
-		Content: v1protocol.Content{
-			ChatID: "testing-adamb",
-			Text:   "abc123",
-		},
-	}
+	s.testMessage = Message{}
```

Comment:
```suggestion
	s.testMessage = Message{
		Text: "abc123",
		ChatId: "testing-adamb",
		ContentType: 0, // should be const
		MessageType: 2, // should be const
		Clock: 154593077368201,
		Timestamp: 1545930773682
	}
```

Reply:
this one would be a bit uglier, as those fields are embedded from `protobuf` so would look something like:
```

s.testMessage = Message{
   ChatMessage: protobuf.ChatMessage{
   .....
   }
}

```

Which one is preferred or there's some other way?

Reply:
Not bad, it's just one struct nested. Up to you. I generally prefer literal. We should definitely replace `0` and `2` with constants.

Reply:
will do, thanks